packages/medusa/oas/admin-spec3-base.yaml

```diff
@@ -231,65 +231,65 @@ tags:
       their sessions.
   - name: App
     description: App endpoints that allow handling apps in Medusa.
-  - name: Batch Job
+  - name: Batch Jobs
     description: Batch Job endpoints that allow handling batch jobs in Medusa.
-  - name: Claim
+  - name: Claims
```

Comment:
Question: Claim endpoints are under `orders` now. Will that change with the generated client to `claims`?

Reply:
I'm fine with that docs-wise. There are other instances for this too including `Discount Conditions` and `Product Variants`. There are also missing tags but I'm not sure if it's important to add them here or not like stock locations.

Reply:
I'm fine with that docs-wise. There are also other tags like Discount Conditions and Product Variants. There are also tags missing from this file like stock location

Reply:
Ok then, I'll do an audit to find endpoints that might not have the appropriate tag. 

As for the tags missing in the base.yaml files, they don't have an impact on our codegen but it would be worth it to add them for the sake of completeness.

Reply:
You can find the list of tags that I updated. 856949eab1485b142418a7814b08415d75efa77c